src/ast/ast.h

```diff
@@ -237,6 +237,7 @@ class AttachPoint : public Node {
   std::string target;
   std::string ns;
   std::string func;
+  usdt_probe_entry usdt; // resolved USDT entry, used to support arguments with wildcard matches
```

Comment:
I couldn't think of another elegant way to read the USDT arguments besides storing them here. Not super happy with this solution as it clutters the definition of an attachpoint, so open to feedback on how to avoid this